src/components/organisms/ContentBox.jsx

```diff
@@ -341,31 +354,34 @@ const FullscreenContainer = styled.div`
 `
 
 const ResponsiveContainer = styled.div`
-  position: absolute;
-  top: 0;
-  bottom: 0;
-  left: 0;
-  right: 0;
+  // position: absolute;
+  // top: 0;
```

Comment:
I'll still this will overwrite with my other code, but since I am not promoting this now, we're okay.